client/app/scripts/liveblog-analytics/styles/analytics.less

```diff
@@ -0,0 +1,137 @@
+@import 'mixins.less';
+@import 'variables.less';
+
+.svg-icon-analytics {
+  background-image: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+PHN2ZyB3aWR0aD0iMjRweCIgaGVpZ2h0PSIyNHB4IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+ICAgICAgICA8dGl0bGU+QXJ0Ym9hcmQ8L3RpdGxlPiAgICA8ZGVzYz5DcmVhdGVkIHdpdGggU2tldGNoLjwvZGVzYz4gICAgPGRlZnM+PC9kZWZzPiAgICA8ZyBpZD0iUGFnZS0xIiBzdHJva2U9Im5vbmUiIHN0cm9rZS13aWR0aD0iMSIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj4gICAgICAgIDxnIGlkPSJBcnRib2FyZCI+ICAgICAgICAgICAgPGcgaWQ9Ikdyb3VwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyLjAwMDAwMCwgMi4wMDAwMDApIj4gICAgICAgICAgICAgICAgPGNpcmNsZSBpZD0iT3ZhbCIgZmlsbD0iIzYyNjI2MiIgY3g9IjIuNSIgY3k9IjE2LjUiIHI9IjIuNSI+PC9jaXJjbGU+ICAgICAgICAgICAgICAgIDxjaXJjbGUgaWQ9Ik92YWwtQ29weSIgZmlsbD0iIzYyNjI2MiIgY3g9IjYuNSIgY3k9IjYuNSIgcj0iMi41Ij48L2NpcmNsZT4gICAgICAgICAgICAgICAgPGNpcmNsZSBpZD0iT3ZhbC1Db3B5LTIiIGZpbGw9IiM2MjYyNjIiIGN4PSIxMy41IiBjeT0iMTMuNSIgcj0iMi41Ij48L2NpcmNsZT4gICAgICAgICAgICAgICAgPGNpcmNsZSBpZD0iT3ZhbC1Db3B5LTMiIGZpbGw9IiM2MjYyNjIiIGN4PSIxNy41IiBjeT0iMi41IiByPSIyLjUiPjwvY2lyY2xlPiAgICAgICAgICAgICAgICA8cGF0aCBkPSJNMiwxOCBDMy42NjY2NjY2NywxNCA1LjMzMzMzMzMzLDEwIDcsNiIgaWQ9IlBhdGgtMiIgc3Ryb2tlPSIjNjI2MjYyIiBzdHJva2Utd2lkdGg9IjEuOCI+PC9wYXRoPiAgICAgICAgICAgICAgICA8cGF0aCBkPSJNNSw1IEM4LjMzMzMzMzMzLDguMzMzMzMzMzMgMTEuNjY2NjY2NywxMS42NjY2NjY3IDE1LDE1IiBpZD0iUGF0aC0zIiBzdHJva2U9IiM2MjYyNjIiIHN0cm9rZS13aWR0aD0iMS44Ij48L3BhdGg+ICAgICAgICAgICAgPC9nPiAgICAgICAgPC9nPiAgICAgICAgPHBhdGggZD0iTTE1LDE3IEwyMCwzIiBpZD0iUGF0aC00IiBzdHJva2U9IiM2MjYyNjIiIHN0cm9rZS13aWR0aD0iMS44Ij48L3BhdGg+ICAgIDwvZz48L3N2Zz4=);
+}
+
+// <svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I left the the original uncompressed svg in there so it could be easily edited. should i drop it somewhere else?